ReactWindows/ReactNative/Chakra/Executor/ChakraJavaScriptExecutor.cs

```diff
@@ -432,9 +444,7 @@ private JavaScriptValue EnsureInvokeFunction()
         {
             if (!_invokeCallbackAndReturnFlushedQueueFunction.IsValid)
             {
-                var globalObject = EnsureGlobalObject();
-                var propertyId = JavaScriptPropertyId.FromString(FBBatchedBridgeVariableName);
-                var fbBatchedBridge = globalObject.GetProperty(propertyId);
+                var fbBatchedBridge = EnsureBatchedBridge();
```

Comment:
Not really possible at this time because the only error channel for the NativeJavaScriptExecutor is an integer at this point.